Formula/zero-install.rb

```diff
@@ -14,77 +23,75 @@ class ZeroInstall < Formula
 
   depends_on "pkg-config" => :build
   depends_on "ocaml" => :build
+  depends_on "ocamlbuild" => :build
   depends_on "opam" => :build
+  depends_on "camlp4" => :build
   depends_on "gtk+" => :optional
-  depends_on :gpg
-
-  # 0install.2.8 resources from https://github.com/ocaml/opam-repository.git
+  depends_on :gpg => :run
```

Comment:
Here specifically or `:gpg` in general? It's only ever really needed during build if upstream has `make check`, etc using `gpg` to test against, or `Makefile`s, etc are checking `gpg` exists before proceeding with installation. Both cases are pretty rare.